Formula/telegraf.rb

```diff
@@ -23,7 +24,7 @@ class Telegraf < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args, "-ldflags", "-X main.version=#{version}", "./cmd/telegraf"
+    system "go", "build", *std_go_args, "-ldflags", "-X github.com/influxdata/telegraf/internal.Version=#{version}-brew", "./cmd/telegraf"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/influxdata/telegraf/internal.Version=#{version}-brew"), "./cmd/telegraf"
```

Reply:
Why the `-s -w`? They don't seem to be added at upstream as well?

Reply:
That's why we generally add them

Reply:
Will profiling then still work?

Reply:
> `-s -w`

it is just for omitting the symbol table and debugging information, has nothing to do with the app.

Could not find the blog post.

Reply:
at this phase, we should make it into the std_go_args and apply by default.

Reply:
> it is just for omitting the symbol table and debugging information, has nothing to do with the app.

It is exactly for this reason the telegraf team didn’t add these flags to their build. Right @jpowers @sspaink?

Reply:
I think you confused the two things, the debugging information for the binary, not for the application it is trying to profiling against?

Reply:
[We also include these](https://github.com/influxdata/telegraf/blob/master/Makefile#L479) when we build release artifacts.